styles.css

```diff
@@ -1,151 +1,108 @@
-
-
-.container
-{
-    display: flex;
-    flex-direction: column;
-   
+.container {
+  display: flex;
+  flex-direction: column;
 }
-.box1{
-  /* My Logo */
```

Comment:
- Take your project to the next level. Kindly, remove unused commented lines and delete blank spaces, that will make your code more readable.